package.json

```diff
@@ -153,5 +153,8 @@
     "transform": [
       "loose-envify"
     ]
+  },
+  "engines": {
+    "node": ">=10.16.3"
```

Comment:
are there any dependencies that we have that need this node version? or else id rather not lock engine break other peoples development environments